### DISCUSSION TITLE
IYP-views beta

### DISCUSSION THREAD

**Discussion by romain-fontugne** [InternetHealthReport/ihr-website#695] [General]
We are currently polishing the web interface for the Internet Yellow Pages (IYP) database. A beta version is available here:
https://exp1.iijlab.net/ihr/en-us/iyp

The goal of this post is to get early feedback so we can fix main problems before we push this to our main website.

Here is a list of things that I already have on my todo list:
- [x]  Integrate better IYP-views and our networks/country reports. One page showing both with a switch to view monitoring widgets or IYP-views
- [x] The top search bar should be replaced with the IYP -views search bar
- [x]  APNIC ranking should start from 1 (not 0). This should be fixed in IYP repo
- [ ] Add a name section for ASes
- [ ] Add a BGP collectors section for ASes
- [ ] Add summary for domain names
- [ ] Add page for authoritative nameservers
- [ ] Reimplement links in tables

**Comment by Arnab0x** [comment]
I can do the second task-

The top search bar should be replaced with the IYP -views search bar

If I am not wrong there is two search bar one at the navbar and the other in the middle. The middle one is working as expected. Only the search bar in the navbar need to be working as the middle one. 

am I right? @romain-fontugne

**Reply by romain-fontugne** [reply]
thanks, but that one requires that we merge the vue3 code first. Please just wait for #669 to be merged

**Reply by dpgiakatos** [reply]
This task will be addressed during the migration of IYP to Vue 3.js.

**Comment by RickDeb2004** [comment]
@romain-fontugne I willing to work on this issue and have done some sort of research on it  **Add a BGP collectors section for ASes**
 Can you bit elaborate on this like ..from where I should start?....
 I have just assumed a basic approach like ...
 If I bring the changes into Asnames.js and add a **getBGPCollector** function which will simulate fetching BGP collectors' information for a given AS number.  Then  a mapping function will be  asynchronously fetching BGP collectors information for each AS and includes it in the result.

**Reply by romain-fontugne** [reply]
no it is not in ASnames.js :D
It is ok, I'll handle that!

**Comment by Animesh239** [comment]
hey there @romain-fontugne .
I am eager to work on the feature :  summary of domain names.
What are the fields of a domain name that should be displayed in the summary?
Registration and Expiration Date
Status(active)
Owner Information
DNS Information
Associated IP Addresses
Social Media Links

**Reply by romain-fontugne** [reply]
we are currently migrating iyp-views to vue.js 3. Let's not add any features for now
